environments/aks/demo.tfvars

```diff
@@ -0,0 +1,18 @@
+cluster_count                      = 2
+kubernetes_cluster_version         = "1.19.11"
+kubernetes_cluster_agent_min_count = "30"
+kubernetes_cluster_agent_max_count = "50"
+kubernetes_cluster_ssh_key         = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDUDkk4BOuQmaj4kO5PEyZ1+HR8u2AzRNkmFkICcQWJakXpNvzec+u1s8nSRaWtuZ8ubQwkTluCHY/OxCdmMOxG3K1+t2Dm0edhPTjGwRuRHzFLawEl8OSvMG97hg3aQMtjlflm05Ao2UCNG1wJLJrkiB5RIu2mBvp1hXolQsbTNUhZLDFDLJYRVoF49EzLbxVwM2jSm1hZESeB+BFgcQJQuEe9ORSldSYqK/c3mw+7EqCw3+zFvkN9fS1z9x2Zg2cnnVCLi/HE6Ul/QDD4TBb/1dFXUEZakXId8oP+W8e/2lTbGbjfW4l3ZnFRyT9B1qO5pQZXAE/CapVOVNOzoG6F"
+sku_tier                           = "Paid"
+enable_user_system_nodepool_split  = true
```

Comment:
Are we enabling for demo?

Reply:
@thomast1906 i have, although happy to set to false if preferred?

Reply:
disabled for now to keep in line with existing Demo cluster.